src/snowflake/snowpark/_internal/udf_utils.py

```diff
@@ -1124,7 +1164,7 @@ def generate_call_python_sp_sql(
     sql_args = []
     for arg in args:
         if isinstance(arg, snowflake.snowpark.Column):
-            sql_args.append(session._analyzer.analyze(arg._expression, {}))
+            sql_args.append(session._analyzer.analyze(arg._expression, {}))  # type: ignore
```

Comment:
mypy things the `_analyzer` object doesn't have an analyze method, but both Analyzer and MockAnalyzer types have the analyze method.